.github/CODEOWNERS

```diff
@@ -16,8 +17,81 @@
+/xml/System.Diagnostics**/** @tommcdon @krwq
+/xml/System.Diagnostics/DiagnosticSource.xml @Anipik
+/xml/System.Diagnostics/EventLog*.xml @Anipik
+/xml/System.Diagnostics/PerformanceCounter*.xml @Anipik
+/xml/System.Diagnostics/Process*.xml @adsitnik @eiriktsarpalis
+/xml/System.Diagnostics/TraceSource.xml @Anipik
+/xml/System.Diagnostics.Tracing**/** @Anipik
+/xml/System.DirectoryServices**/** @tquerec @josephisenhour
+/xml/System.Drawing**/** @JeremyKuhne @maryamariyan
+/xml/System.Dynamic**/** @cston @333fred
+/xml/System.Globalization**/** @safern @krwq @tarekgh
+/xml/System.IO.Compression**/** @carlossanlop @ericstj
+/xml/System.IO.Pipelines**/** @davidfowl @halter73 @jkotalik @anurse
+/xml/System.IO**/** @JeremyKuhne @carlossanlop @jozkee
+/xml/System.Linq**/** @eiriktsarpalis
+/xml/System.Linq/Parallel*.xml @tarekgh @kouvel
+/xml/System.Linq.Expressions**/** @cston @333fred
+/xml/System.Management**/** @Anipik
+/xml/System.Net**/** @dotnet/ncl
+/xml/System.Numerics**/** @tannergooding @prgovind
+/xml/System.Reflection**/** @steveharter @GrabYourPitchforks
+/xml/System.Reflection.Emit**/** @steveharter @GrabYourPitchforks
+/xml/System.Reflection.Metadata**/** @tmat @nguerrera
+/xml/System.Resources**/** 	@buyaa-n @tarekgh @krwq
+/xml/System.Runtime**/** @bartonjs @joperezr
+/xml/System.Runtime.Caching**/** @StephenMolloy @HongGit
+/xml/System.Runtime.CompilerServices**/** @Anipik @steveharter
+/xml/System.Runtime.InteropServices**/** @AaronRobinsonMSFT @jkoritzinsky
+/xml/System.Runtime.Intrinsics**/** @tannergooding @CarolEidt @RussKeldorph
+/xml/System.Security**/** @bartonjs @GrabYourPitchforks @krwq
+/xml/System.ServiceModel.Syndication**/** @StephenMolloy @HongGit @mconnew
+/xml/System.ServiceProcess**/** @Anipik
+/xml/System.Text**/** @tarekgh @GrabYourPitchforks
+/xml/System.Text/Encoding*.xml @layomia @krwq @tarekgh
+/xml/System.Text.Encodings.Web**/** @GrabYourPitchforks @layomia @tarekgh
+/xml/System.Text.Json**/** @layomia @steveharter @jozkee
+/xml/System.Text.RegularExpressions**/** @prgovi @eerhardt
+/xml/System.Threading**/** @kouvel
+/xml/System.Threading.Channels**/** @tarekgh @stephentoub
+/xml/System.Threading.Tasks**/** @tarekgh @stephentoub
+/xml/System.Transactions**/** @dasetser @HongGit
+/xml/System.Web**/** @scottaddie
+/xml/System.Windows.Forms**/** @RussKie
+/xml/System.Windows**/** @vatsan-madhavan
+/xml/System.Xaml**/** @vatsan-madhavan
+/xml/System.Xml**/** @buyaa-n @krwq
+
+# System.Console
+/xml/System/Console** @carlossanlop @eiriktsarpalis
+
+# System.Memory
+/xml/System/Memory*.xml @GrabYourPitchforks @adsitnik
+/xml/System/ReadOnlyMemory*.xml @GrabYourPitchforks @adsitnik
+/xml/System/ReadOnlySpan*.xml @GrabYourPitchforks @adsitnik
+/xml/System/Span*.xml @GrabYourPitchforks @adsitnik
+
+# System.Runtime
+/xml/System/Array.xml @bartonjs @joperezr
+
+# System.Uri
+/xml/System/Uri.xml @dotnet/ncl
+
+/xml/Microsoft.CSharp**/** @cston @333fred
 /xml/Microsoft.SqlServer**/** @stevestein
```

Comment:
The owner of Microsoft.SqlServer is not mentioned in the area owners document, but I'll keep it there.

Reply:
@stevestein are you still the correct person to be flagged when there are PR's that edit the documentation for Microsoft.SqlServer.

Reply:
You probably won't get an answer here. Keep as-is for now.